chart/templates/_helpers.yaml

```
@@ -731,85 +731,148 @@ server_tls_key_file = /etc/pgbouncer/server.key
+    +-----------------------------+      +------------------------+      +----------------------+      +-----------------+      +-------------------------+
 
-Values are not accumulated meaning that if runAsUser is set to 10 in <node>.securityContext,
+Values are not accumulated meaning that if runAsUser is set to 10 in <node>.securityContexts.pod,
 any extra values set to securityContext or uid+gid will be ignored.
 
 The template can be called like so:
-   include "airflowSecurityContext" (list . .Values.webserver)
+   include "airflowPodSecurityContext" (list . .Values.webserver)
 
 Where `.` is the global variables scope and `.Values.webserver` the local variables scope for the webserver template.
 */}}
-{{- define "airflowSecurityContext" -}}
-  {{- $ := index . 0 }}
+{{- define "airflowPodSecurityContext" -}}
+  {{- $ := index . 0 -}}
   {{- with index . 1 }}
-    {{- if .securityContext }}
-      {{- toYaml .securityContext }}
-    {{- else if $.Values.securityContext }}
-      {{- toYaml $.Values.securityContext }}
-    {{- else }}
+    {{- if .securityContexts.pod -}}
+{{ toYaml .securityContexts.pod | print }}
+    {{- else if .securityContext -}}
+{{ toYaml .securityContext | print }}
+    {{- else if $.Values.securityContexts.pod -}}
+{{ toYaml $.Values.securityContexts.pod | print }}
+    {{- else if $.Values.securityContext -}}
+{{ toYaml $.Values.securityContext | print }}
+    {{- else -}}
 runAsUser: {{ $.Values.uid }}
 fsGroup: {{ $.Values.gid }}
     {{- end }}
   {{- end }}
 {{- end }}
 
 {{/*
-Set the default value for securityContext
-If no value is passed for securityContext or <node>.securityContext, defaults to UID in the local node.
+Set the default value for pod securityContext
+If no value is passed for <node>.securityContexts.pod or <node>.securityContext, defaults to UID in the local node.
 
-    +------------------------+     +-------------+
-    | <node>.securityContext |  >  | <node>.uid  |
-    +------------------------+     +-------------+
+    +-----------------------------+      +------------------------+      +-------------+
+    | <node>.securityContexts.pod |  ->  | <node>.securityContext |  ->  | <node>.uid  |
+    +-----------------------------+      +------------------------+      +-------------+
 
 The template can be called like so:
-  include "localSecurityContext" .Values.statsd
+  include "localPodSecurityContext" (list . .Values.schedule)
 
 It is important to pass the local variables scope to this template as it is used to determine the local node value for uid.
 */}}
-{{- define "localSecurityContext" -}}
-  {{- if .securityContext }}
-    {{- toYaml .securityContext }}
-  {{- else }}
-    {{- printf "runAsUser: %v" .uid }}
-  {{- end }}
-{{- end }}
+{{- define "localPodSecurityContext" -}}
+  {{- if .securityContexts.pod -}}
+{{ toYaml .securityContexts.pod | print }}
```

```suggestion
    {{- toYaml .securityContexts.pod | print }}
```

Here too. Last one I'll comment on, but hit the rest of these as well?